permissions/plugin-dashboard-view.yml

```diff
@@ -1,7 +1,8 @@
 ---
 name: "dashboard-view"
-github: "jenkinsci/dashboard-view-plugin"
+github: &gh "jenkinsci/dashboard-view-plugin"
 issues:
+  - github: *gh
   - jira: '15679'  # dashboard-view-plugin
```

Comment:
recommend adding report: false if you don't want people to use jira anymore

https://github.com/jenkins-infra/repository-permissions-updater#declaring-issue-trackers

Reply:
We can also just archive the component.

Reply:
yea but report: false removes the "report an issue (jira)" from plugin site